### ISSUE TITLE
addValues function typo , result.causes should be result.clauses?

### ISSUE THREAD

**Issue by rajansen** [FrankyBoy/jasmine-params#1]
Tip to itterate through an array:

```
jparams
     .iterate('useCase')
     .values.apply(this,useCases).
     describe('test ', function (useCase) {.....
```

**Comment by FrankyBoy** [comment]
Thanks for pointing that out. I do not use the package any more so I missed that.

**Comment by FrankyBoy** [comment]
Published 0.3.2 to npm
